src/constants/addresses.ts

```diff
@@ -5,6 +5,10 @@ type AddressMap = { [chainId: number]: string }
 
 export const UNI_ADDRESS: AddressMap = constructSameAddressMap('0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984')
 
+// celo v3 addresses
+const CELO_ROUTER_ADDRESS = '0x5615CDAb10dc425a742d643d949a7F474C01abc4'
```

Comment:
Just inline these to keep this file simpler, even though they'll be repeated for CELO and CELO_ALFAJORES.